ngraph/core/reference/include/ngraph/runtime/reference/topk.hpp

```diff
@@ -80,7 +80,7 @@ namespace ngraph
                       size_t axis,
                       size_t k,
                       bool compute_max,
-                      op::TopK::SortType sort = op::TopK::SortType::NONE)
+                      op::v1::TopK::SortType sort = op::v1::TopK::SortType::NONE)
```

Comment:
Maybe will be better to use `TopKSortType` instead of versioned `SortType`

Reply:
kmb-plugin depends on op::v1::TopK::SortType, so I guess we should stick to that for now.